src/SummaryRow.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import SummaryCell from './SummaryCell';
+import { SummaryRendererProps } from './common/types';
+
+export default function SummaryRow<R>({
+  extraClasses,
+  height,
+  rowIdx,
+  lastFrozenColumnIndex,
+  row,
+  scrollLeft,
+  viewportColumns,
+  width
+}: SummaryRendererProps<R>) {
+  return (
+    <div
+      className={classNames(
+        'rdg-row',
+        `rdg-row-${rowIdx % 2 === 0 ? 'even' : 'odd'}`,
+        extraClasses
```

Comment:
Is `extraClasses` being used?

Reply:
hm... I thought it was used for passing the extra class directly to the row, but looks like even for `Row` component it's a dead props as well. Do we want to align this props together in a new PR?

Reply:
I think we can remove it. `extraClasses` is set using `rowRenderer` but for summary rows we do not use the `rowRenderer`. This is the only place we call it

https://github.com/adazzle/react-data-grid/pull/1921/files#diff-6ee01b9be15a1d3a39e71ca4ea0d1a91R214